R/bootstrap.R

```diff
@@ -1342,13 +1343,15 @@ uiOutput <- htmlOutput
 #' @export
 downloadButton <- function(outputId,
                            label="Download",
-                           class=NULL, ...) {
+                           class=NULL,
+                           icon=icon("download"),
```

Comment:
Can you put `icon` after the `...`? That would reduce the risk of breaking existing code, and it also is a pattern we are using more and more in general (forcing users to name the arguments).

Reply:
Done ✅